src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/AvoidEmptyInterfaces.Fixer.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.CodeQuality.Analyzers.ApiDesignGuidelines
     /// </summary>
     public abstract class AvoidEmptyInterfacesFixer : CodeFixProvider
     {
-        public sealed override ImmutableArray<string> FixableDiagnosticIds => ImmutableArray.Create(AvoidEmptyInterfacesAnalyzer.RuleId);
+        public sealed override ImmutableArray<string> FixableDiagnosticIds => ImmutableArray<string>.Empty;
```

Comment:
:\ These are going to be easy to mess up if/when the fixers are implemented. I would _strongly_ prefer to update the analysis to recognize the stub implementation in `RegisterCodeFixesAsync` (returning a completed task) and use that as the trigger for True/False.

Reply:
The [current approach](https://github.com/dotnet/roslyn-analyzers/blob/master/src/GenerateAnalyzerRulesets/Program.cs#L448) to generate the documentation files does not to source analysis. It loads the analyzer assemblies via our AnalyzerFileReference API and then uses the runtime object to query supported diagnostics (analyzer) and fixable diagnostics (fixer). It is unlikely that this will regress because the changed files were all auto-generated stubs by a tool, and I don't think we will change these again unless we are actually implementing a fixer.

Reply:
Would be neat to come up with something, but this seems fine to get the docs updated before shipping 2.9.5-beta1.

Reply:
Yep, we can have a separate discussion on whether we need a better approach here. I am going to retain the current approach for this PR.